### PR TITLE
Glide cleanup

### PR DESCRIPTION
**Pull request by mrIncompetent**
What was the issue:
Our glide.yaml manually defined all dependencies of kubernetes.
Since Glide is able to read and process the godep.json of kubernetes when we only define kubernetes alone as dependency our gilde.yaml was bigger than it should be and tried to manage dependencies of other project.
When we updated kubernetes from 1.3 -> 1.4 we did not updated the kubernetes dependencies in our own glide.yaml. This caused conflicts since we overwrite this way the versions of kubernetes dependencies.

With this pr we only define our own dependencies in the glide.yaml. If we have a shared dependency with kubernetes or so, we do specify it in the glide.yaml but we dont specify a version.

Please test if the oauth/jwt stuff still works, since i currently do not have a running local dev-env

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -34,10 +34,9 @@ import:
   - signer
   - signer/local
 - package: github.com/dgrijalva/jwt-go
-  version: 5ca80149b9d3f8b863af0e2bb6742e608603bd99
 - package: github.com/digitalocean/godo
   version: 6ca5b770f203b82a0fca68d0941736458efa8a4f
-- package: github.com/ghodss/yaml
+- packagUsere: github.com/ghodss/yaml
```
Comment (scruplelesswizard) [inline glide.yaml]: This doesn't look right?

File: handler/user.go [hunk 2]
```diff
@@ -23,15 +23,20 @@ func decodeUserReq(r *http.Request) (interface{}, error) {
 
 	obj := context.Get(r, "user")
 	token := obj.(*jwt.Token)
-	var ok bool
-	req.user.Name, ok = token.Claims["sub"].(string)
+
+	claims, ok := token.Claims.(jwt.MapClaims)
+	if !ok || !token.Valid {
+		return nil, fmt.Errorf("Invalid token in JWT in request %v", r)
```
Comment (scruplelesswizard) [inline handler/user.go]: Redundant statement. 
`"Invalid JWT in request %v"`

**Review by scruplelesswizard — CHANGES_REQUESTED** [review]
A few nits. 

Also, for future, please make code changes and vendor changes in separate commits (`git add vendor/ && git commit -m "Updated vendoring" && git add . && git commit` )
//cc @realfake

File: glide.yaml (hunk 1 above)
Comment (mrIncompetent) [reply glide.yaml]: You're right. No idea how it came there...
Fixed with 004b1d4

File: handler/user.go (hunk 2 above)
Comment (mrIncompetent) [reply handler/user.go]: Fixed with b2cca25

**Review by scruplelesswizard — APPROVED** [review]
LGTM

**Comment by scheeles** [comment]
LGTM

**Pull request merged**